tokio-tls/examples/echo.rs

```diff
@@ -0,0 +1,59 @@
+// A tiny async TLS echo server with Tokio
+extern crate native_tls;
+extern crate tokio;
+extern crate tokio_tls;
+
+use native_tls::Identity;
+use tokio::io;
+use tokio::net::TcpListener;
+use tokio::prelude::*;
+
+fn main() {
+    // Bind the server's socket
+    let addr = "127.0.0.1:12345".parse().unwrap();
+    let tcp = TcpListener::bind(&addr).unwrap();
+
+    // Create the TLS acceptor.
+    let der = include_bytes!("identity.p12");
```

Comment:
This file is missing in repo

Reply:
I used that file cause it is in the repo, at least in master.

https://github.com/tokio-rs/tokio/tree/master/tokio-tls/examples

Reply:
Sorry, I missed it